user/src/components/FormList/FormList.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import Button from '@/components/Button';
+import FormContainer from '@/components/FormContainer';
+import { FormItem } from './type';
+
+type FormListProps = {
+  items: FormItem[];
+};
```

Comment:
ボタン押下時のコールバックを追加すると楽かなと思いました。
```suggestion
type FormListProps = {
  items: FormItem[];
  onEdit?: () => void;
};
```
```
<Button size="pc" color="main" type="button" icon="pencil" onClick={onEdit}>
```

Reply:
確かに！
ついでに締め切った後はリストの実になるからボタン表示非表示もpropsで管理させます！